README.md

```diff
@@ -18,6 +18,30 @@ git clone https://github.com/redhat-et/copilot-ops.git
 # Build the binary
 make build
 ```
+To install the command line tool, complete the following steps:
+
+```console
+# Create bin directory
+mkdir -p ${HOME}/bin
+
+# Save binary to bin directory (run from copilot-ops directory)
+cp copilot-ops ${HOME}/bin
+
+# Make binary executable 
+chmod 755 ${HOME}/bin/copilot-ops
+
+# Open file ${HOME}/.bashrc in text editor, add line and save
+export PATH= "${HOME}/bin:${PATH}"
+
+# Rerun .bashrc file 
+. ~/.bashrc
+
+# Ensure binary is on PATH
+command -v copilot-ops
```

Comment:
We can omit this since the user will know that it's not successful if it breaks when running the next command

Reply:
Line removed